configs/example/apu_se.py

```diff
@@ -533,10 +540,16 @@
                 simd_id=j, wf_size=args.wf_size, num_regs=args.sreg_file_size
             )
         )
+        rfcs.append(
+            RegisterFileCache(
+                simd_id=j, cache_size=args.register_file_cache_size
+            )
+        )
```

Comment:
Should this be added to the FS mode config too? (configs/example/gpufs/system/amdgpu.py I think).

Reply:
Good point.  @KaiBatley can you please do this?